core/src/main/kotlin/org/evomaster/core/problem/api/service/ApiWsStructureMutator.kt

```diff
@@ -46,6 +47,12 @@ abstract class ApiWsStructureMutator : StructureMutator() {
     @Inject
     protected lateinit var harvestResponseHandler: HarvestActualHttpWsResponseHandler
 
+    // TODO: This should only be initialized when config.generateSqlDataWithDSE is enabled
+    // TODO: Also, z3solver.close() should be invoked
+    // when the application is shutting down to stop the Docker container and clean the tmp folder
+    @Inject
```

Comment:
@arcuri82 How and where should I do that?

Reply:
@agusaldasoro read `docs/for_developers.md`, in particular the part about `Guice`. So, likely would need to be defined in `org.evomaster.core.BaseModule`